retriever/bin/retriever.js

```diff
@@ -30,9 +36,11 @@ export default {
    * @param {Env} env
    * @param {ExecutionContext} ctx
    * @param {typeof defaultRetrieveFile} retrieveFile
+   * @param {object} options
+   * @param {AbortSignal} [options.signal]
    * @returns
    */
-  async _fetch(request, env, ctx, retrieveFile) {
+  async _fetch(request, env, ctx, retrieveFile, { signal } = {}) {
```

Comment:
Can we use the same params as in the public-facing `fetch` method?

```suggestion
  async _fetch(request, env, ctx, {retrieveFile, signal }) {
```